### ISSUE TITLE
IX-F import deletes data and I don't get it

### ISSUE THREAD

**Issue by arnoldnipper** [peeringdb/admincom#151]
See https://peeringdb.com/cp/peeringdb_server/ixlanixfmemberimportlog/171/change/, but

(server0:nipper 38 ) curl -sG https://portal.de-cix.net/customer_mad | jq -c '.member_list[] | select (.asnum==13335) | [ .asnum, .connection_list[].vlan_list[].ipv4.address, .connection_list[].vlan_list[].ipv6.address]'

[13335,"185.1.68.12","2001:7f8:a0::3417:0:1"]

(server0:nipper 39 ) curl -sG https://peeringdb.com/api/netixlan --data-urlencode ix_id=1277 | jq -c '.data[] | select( .asn==13335 ) | [ .asn, .ipaddr4, .ipaddr6 ]'  
                              
[13335,"185.1.68.12","2001:7f8:a0::3417:0:1"]

There is **no** difference between both records afais

**Comment by arnoldnipper** [comment]
Tool only looks for "active" and "connected"

**Comment by arnoldnipper** [comment]
@vegu, is it worth to consider to allow values as listed in https://github.com/euro-ix/json-schemas/wiki/Schema-Field-Entries-Members ? All of these states do mean that the network is a member of the IX.

**Comment by vegu** [comment]
@arnoldnipper You are correct that they are technically all members of an ix, however from a developer's standpoint i'd add that in PeeringDB we currently have no way to accurately describe any state but `connected`/`active` (status `ok` on our end)

More of a PeeringDB Policy question i think? As in what entries does PDB want to enter it's database and if we let all/most through, do we accurately describe in peeringdb to the same granularity as the ixf schema?

I'd raise the point on the main pdb github.

**Comment by arnoldnipper** [comment]
@vegu, my point is not to track any state, but that PDB accepts any state listed in the URL mentioned as "good" (i.e. this network is a member of the IX).

There must have been a reason for only accepting `connected / active`. Which one was that as `active` is not even listed.

**Comment by vegu** [comment]
> There must have been a reason for only accepting connected / active. Which one was that as active is not even listed.

While the schema suggests possible values, the schema itself does not force/validate state from a set of choices - `active` was found in a bunch of urls we tested against during implementation, so we chose to support it.

The reason to limit to `connected`/`active` was simply that it was unclear on what peeringdb was to do with the other states, if anything at all. If it's appropriate to have peeringdb just import all states as `ok` then that's certainly something we can do fairly easily.
